src/hypergol/dataset.py

```diff
@@ -1,21 +1,16 @@
 import os
 import glob
-import gzip
-import json
-import hashlib
 
-from datetime import datetime
 from pathlib import Path
 
+from hypergol.datachunk import DataChunk
 from hypergol.utils import Repr
-from hypergol.base_data import BaseData
+from hypergol.utils import _get_hash
```

Comment:
Importing a private function from another file? Something is weird here

Reply:
yeah, should have been refactored to be public